src/orquestra/sdk/_base/abc.py

```diff
@@ -194,6 +194,12 @@ def get_workflow_logs(
         """
         raise NotImplementedError()
 
+    def get_workflow_project(self, wf_run_id: WorkflowRunId):
```

Comment:
```suggestion
    @abstractmethod
    def get_workflow_project(self, wf_run_id: WorkflowRunId) -> ProjectRef:
```

Reply:
Looks like we forgot to sprinkle `@abstractmethod` on other methods in this interface too.